tests/test_wave_07_opt.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+
+def test_items_have_age_as_int():
+    items = [
+        Clothing(age=3),
+        Decor(age=3),
+        Electronics(age=3)
+    ]
+    for item in items:
+        assert item.age == pytest.approx(3)
+
+
+
+def test_get_newest():
+    item_a = Clothing(age=2)
+    item_b = Decor(age=3)
+    item_c = Clothing(age=4)
+    item_d = Decor(age=5)
+    item_e = Clothing(age=3)
+    tai = Vendor(
+        inventory=[item_a, item_b, item_c, item_d, item_e]
+    )
+
+    newest_item = tai.get_newest()
+
+    assert newest_item.category == "Clothing"
+    assert newest_item.age == pytest.approx(2)
+
+
+
+def test_swap_by_newest():
```

Comment:
You might also add a test for:

1.  What happens if one or more vendors do not have any items
2. What happens if a vendor has two items of the same age